topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,743 @@
+>
+> Workflows default to requiring an AF ≥ 0.05 and AV-supporting reads of ≥ 10 (these and all other parameters can be easily changed by the user). For an AV to be listed in the reports it must surpass these thresholds in at least one sample of the respective dataset. We estimate that for AV calls with an AF ≥ 0.05 our analyses have a false-positive rate of < 15% for both Illumina RNAseq and Illumina ARTIC data, while the true-positive rate of calling such low-frequency AVs is ~80% and approaches 100% for AVs with an AF ≥ 0.15. This estimate is based on an initial application of the Illumina RNAseq and Illumina ARTIC workflows to two samples for which data of both types had been obtained at the virology department of the University of Freiburg and the assumption that AVs supported by both sets of sequencing data are true AVs. The second threshold of 10 AV-supporting reads is applied to ensure that calculated AFs are sufficiently precise for all AVs.
+>
+{: .details}
+
+> ### {% icon hands_on %} Hands-on: From FASTQ to annotated AVs
+>
+> 1. **Get the workflow** on Galaxy
+>    
+>    - Option 1: Use workflows available on [usegalaxy.org](https://usegalaxy.org/), [usegalaxy.eu](https://usegalaxy.eu/) or [usegalaxy.org.au](https://usegalaxy.org.au/)
+>
+>      Click on the appropriate workflow version (Illumina RNAseq PE for the example datasets)
+>
+>        &#8595; Type of data / Galaxy instance &#8594; | [usegalaxy.org](https://usegalaxy.org/) | [usegalaxy.eu](https://usegalaxy.eu/) | [usegalaxy.org.au](https://usegalaxy.org.au/)
+>        --- | --- | --- | ---
+>        Illumina ARTIC PE (*to use for example datasets*) | [{% icon workflow %}](https://usegalaxy.org/workflows/run?id=d4ea6cdd40522eb1) | [{% icon workflow %}](https://usegalaxy.eu/workflows/run?id=2f9fa06b1a927a07) | [{% icon workflow %}](https://usegalaxy.org.au/workflows/run?id=7fd58f5fc93f414e)
+>        Illumina RNAseq PE | [{% icon workflow %}](https://usegalaxy.org/workflows/run?id=5a610d5a42d50cf3) | [{% icon workflow %}](https://usegalaxy.eu/workflows/run?id=e7ba6ca41d46baf2) | [{% icon workflow %}](https://usegalaxy.org.au/workflows/run?id=bd1cf22d47389742)
+>        Illumina RNAseq SE | [{% icon workflow %}](https://usegalaxy.org/workflows/run?id=c092a3631d68ce38) | [{% icon workflow %}](https://usegalaxy.eu/workflows/run?id=6b41f7afbe14647d) | [{% icon workflow %}](https://usegalaxy.org.au/workflows/run?id=31dbd313e5c8160b)
+>        ONT ARTIC | [{% icon workflow %}](https://usegalaxy.org/workflows/run?id=88d0b64011c3148c) | [{% icon workflow %}](https://usegalaxy.eu/workflows/run?id=17d7f9bddbc834f8) | [{% icon workflow %}](https://usegalaxy.org.au/workflows/run?id=0c6f7bfc826433c4)
+>
+>      The browser will open a new tab with Galaxy's workflow invocation interface.
+>
+>    - Option 2: Import the workflow
+>
+>      - Copy the URL (e.g. via right-click) of the appropriate workflow version or download it to your computer:
```

Comment:
Should we just include the workflowhub import? or use the IWC URLs?

Reply:
Updated